src/Toml/PrefixTree.hs

```diff
@@ -83,7 +83,33 @@ singleT = Leaf
 
 -- | Inserts key-value element into the given 'PrefixTree'.
 insertT :: Key -> a -> PrefixTree a -> PrefixTree a
-insertT = undefined
+insertT newK newV (Leaf k v) =
+    case keysDiff k newK of
+        Equal -> Leaf k newV
+        NoPrefix -> error "Impossible happened"
```

Comment:
Better to write something like `"Algorithm error: can't be equal prefixes in insertT:Leaf case"`.